caluma/caluma_core/tests/test_health_checks.py

```diff
@@ -0,0 +1,542 @@
+import pytest
+
+from .. import models
+from .fake_model import get_fake_model
+from .fake_model import define_fake_model
+from ..health_checks import check_migrations
+from django.db.models import CharField
+from django.test import Client
+import django.db
+from django.db import connections
+from django.db.backends.signals import connection_created
+from django.urls import reverse, resolve
+from django.conf import settings
+import time
+from urllib3.response import HTTPResponse
+from django.test import override_settings
+import xml
+
+from caluma.caluma_form import storage_clients
+from django.db.migrations.executor import MigrationExecutor
+
+from minio import Minio
+from minio.definitions import Object as MinioStatObject
+from minio import ResponseError
+
+from django.core import management
+
+import sys
```

Comment:
Hmm, don't we have `isort` checking anymore? Anyway, please run your code through `isort` to clean up your imports